### DISCUSSION TITLE
打开颜色集，提示文件不存在或格式异常

### DISCUSSION THREAD

**Discussion by YeYeYeYeYe** [WatsonYao/nian_app#95] [Q&A] [answered]
刚开始用，也没什么需要备份到，卸载删除了几次文件夹重新安装也没用，目前不敢继续用怕后面数据异常无法备份
我应该怎么操作修复这个呀？

![微信图片_20241013173123](https://github.com/user-attachments/assets/d9a26ae9-08bc-4b9d-9130-1e501c40b52a)

**Comment by WatsonYao** [comment, marked as answer]
帮助文档里面有介绍的，可参考下。
https://docs.qq.com/doc/DZkVOelRPbkhUZHJB 
<img width="276" alt="截屏2024-10-14 08 56 34" src="https://github.com/user-attachments/assets/925a0115-b865-416d-8583-b426089e6d2d">
